### DISCUSSION TITLE
Voice conversion models

### DISCUSSION THREAD

**Discussion by AlessandroOrsatti** [coqui-ai/TTS#3334] [General Q&A]
Hi everybody, where can I find the list of all the available models for Voice Conversion? For now I only found "voice_conversion_models/multilingual/vctk/freevc24" which is the one listed on the readme example for VC, TTS.list_models() only prints tts_models and not voice_conversion_models

**Comment by schokopfannkuchen** [comment]
apparently there is only one model

```shell
tts --list_models
``` 

outputs [trimmed]:
```
 Name format: type/language/dataset/model
 1: voice_conversion_models/multilingual/vctk/freevc24
```

**Comment by curita2023** [comment]
Is this your folder?
No,it isn’t. 

Is this your father?
yes,it is.

Is this your mother?
No,it’s not.
